pandas/core/strings.py

```diff
@@ -241,6 +252,117 @@ def g(x):
         return lib.map_infer(arr, f)
 
 
+def str_format(
+    arr,
+    format: str,
+    name: Optional[str] = None,
+    positional_only: bool = False,
+    how_na: str = "any",
+) -> "Series":
+    """
+    Format rows according to the format and return a Series with one string per row.
+
+    Parameters
+    ----------
+    arr : DataFrame or Series
+        The values to format.
+    format : str
+        format string.
+    name : Label, optional
+        The name of the returned Series.
+    positional_only : bool, default False
```

Comment:
do we actually need this parameter? isn't positional obvious if you don't have field labels?